tools/cldr-apps/js/src/index.js

```diff
@@ -8,7 +8,7 @@ import "../../../cldr-code/src/main/resources/org/unicode/cldr/tool/reports.css"
 
 // module stylesheets need to go here. See cldrVueRouter.js
 // example: import 'someModule/dist/someModule.css'
-import "ant-design-vue/dist/antd.css";
+import "ant-design-vue/dist/antd.min.css";
```

Comment:
This change, per https://github.com/ant-design/ant-design/issues/33327#issuecomment-1046404732 , cut the bundle.js size in *half* from 6MB to 3MB.